### ISSUE TITLE
Allow aliases for grouping

### ISSUE THREAD

**Issue by hauleth** [git-chglog/git-chglog#181]
I have project where the tag for feature changed from `feat` to `ft`. Now when I generate the changelog I have duplicated entry of "Features". It would be handy to have a way to group them together as one.

## Expected Behavior

```
$ git log --oneline
* <sha> ft: foo
* <sha> feat: bar
$ git chglog
[…]

Features:
* foo

Features:
* bar
```

## Actual Behavior

```
$ git log --oneline
* <sha> ft: foo
* <sha> feat: bar
$ git chglog
[…]

Features:
* foo
* bar
```

**Comment by MaxenceG2M** [comment]
Hi hauleth,

You may have reversed "Actual Behavior" and "Expected Behavior", isnt' you?

I think I have the same problem.
I try to "override" with other aliases but chglog keep more than one entry for a same group.

**Comment by hauleth** [comment]
@MaxenceG2M indeed, fixed it now.

**Comment by tprestegard** [comment]
Agreed, I would like to be able to map commits starting with either `doc:` or `docs:` to a single section.

**Comment by rshmhrj** [comment]
@hauleth @MaxenceG2M @tprestegard 

Please see this workaround I've put together and lmk if it helps:

https://github.com/git-chglog/git-chglog/issues/33#issuecomment-1897789708

**Comment by hauleth** [comment]
I have moved from `git-chglog` with `git-cliff` and it has everything I need. I do not look back.
